### DISCUSSION TITLE
Color theme seems bugged

### DISCUSSION THREAD

**Discussion by AlejandroSanchez90** [dandavison/delta#1912] [General]
the updated change color green is too dark, is this the intended behavior?
![image](https://github.com/user-attachments/assets/e1f90443-4e72-46de-825f-386b515dc6a3)

**Comment by dandavison** [comment]
It's possible that your terminal does not support "truecolor" aka 24-bit color. Can you try with `true-color = false`, and try a truecolor terminal test such as https://gist.github.com/lifepillar/09a44b8cf0f9397465614e622979107f?
